.github/workflows/trigger_pull_request.yml

```diff
@@ -0,0 +1,16 @@
+name: Slash Command Dispatch Pull Request Trigger
```

Comment:
what is the slash command ? maybe include that in the name - `/prepare-release Dispatch Pull Request Trigger`

Reply:
At first I thought this was a Slack command. Maybe call this `/prepare-release Dispatch Pull Request Comment Trigger`

Reply:
Sound good i could be more clear with this